pyp5js/compiler.py

```diff
@@ -38,10 +38,10 @@ def run_compiler(self):
         Execute transcrypt command to generate the JS files
         """
         command = self.command_line
-        cprint.info(f"Converting Python to P5.js...\nRunning command:\n\t {command}")
+        cprint.info(
+            f"Converting Python to P5.js...\nRunning command:\n\t {command}")
 
-        proc = subprocess.Popen(shlex.split(command))
-        proc.wait()
+        os.system(command)
```

Comment:
What do you think about trying out `subprocess.call` method? I was reading [`os.system` docs](https://docs.python.org/3/library/os.html#os.system) and it says it would be better to use the `subprocess` module instead of `os.system`. The docs [links to an example](https://docs.python.org/3/library/subprocess.html#replacing-os-system) replacing it by the `subprocess.call` method.